examples/sign/connector-sign-sync.yml

```diff
@@ -11,6 +11,16 @@ sign_orgs:
     # Sign account associated with admin_email value will be skipped
     admin_email: "Admin user associated with key e.g. user@example.com"
 
+connection:
+  # Number of allowed concurrent requests (higher is faster, but consumes more bandwidth and memory)
+  request_concurrency: 5
+  # Number of requests to queue at one time.  Reduce if memory usage is too high.
+  batch_size: 10000
+  # Number of times to retry failed requests
+  retry_count: 5
+  # Timeout for requests in seconds
+  timeout: 120
+  
```

Comment:
I don't know if it makes sense to document these options. We should make them available, but I think the defaults will work for most customers. What do you think about making them undocumented? We may mention them in documentation in the future, but for now we can work with individual customers to tweak these settings if they experience problems.